templates/base.html

```diff
@@ -183,10 +181,5 @@
     <script>$.fn.dataTable.ext.order.intl('{{ LANGUAGE_CODE }}');</script>
 
     {% block script %}{% endblock %}
-    <!-- Mathjax / LaTeX support-->
```

Comment:
Are you sure we don't need Mathjax? Isn't this used by same workshop pages to display latex?

Reply:
I've never seen it actually used and didn't even know this feature existed until I saw this - the new editor doesn't even have a button for it (but we can add it back if you think it's necessary)

Reply:
Hmm... let's skip it. If something not important is broken, then ine still can understand the formulas. If something important is broken, then we will readd it.